.github/workflows/codeql-analysis.yml

```
@@ -0,0 +1,81 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ master, hotfix/**/*, release/**/* ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ master ]
```

This takes a long time to run. I think we can be happy with just running on merges to master/hotfix (and `release/1.x` is deprecated)

```suggestion
on:
  push:
    branches: [ master, hotfix/**/* ]
```